Stack/Opc.Ua.Core/Stack/State/DataTypeState.cs

```diff
@@ -191,6 +191,7 @@ public override void Update(ISystemContext context, BinaryDecoder decoder, Attri
                     if (ServiceResult.IsGood(result))
                     {
                         if (dataTypeDefinition?.Body is StructureDefinition structureType &&
+                            structureType.DefaultEncodingId != null &&
```

Comment:
I think it should be rather DefaultEncodingId  == null || DefaultEncodingId.IsNullNodeId ? otherwise it is never set?